### PR TITLE
feat!: update emojis to match yat emojis

### PR DESCRIPTION
**Pull request by SWvheerden**
Description
---
This updates the emoji_id emojis to match those used by yat. 

Motivation and Context
---
This will allow users to create a yat from their emoji_id as they now share all characters. 

How Has This Been Tested?
---
unit tests

Breaking Changes
---

- [] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [x] Other - Please specify

This causes the string emoji representation of tari addresses to be incompatible with previous versions.

### CODE REVIEW

**Review by AaronFeickert — DISMISSED** [review]
utACK

**Comment by AaronFeickert** [comment]
This may give merge conflicts with #6287, so beware!

File: base_layer/common_types/src/emoji.rs [hunk 1]
```diff
@@ -78,20 +78,20 @@ const CHECKSUM_SIZE: usize = 1; // number of bytes in the checksum
 
 // The emoji table, mapping byte values to emoji characters
 pub const EMOJI: [char; DICT_SIZE] = [
-    '🌀', '🌂', '🌈', '🌊', '🌋', '🌍', '🌙', '🌝', '🌞', '🌟', '🌠', '🌰', '🌴', '🌵', '🌷', '🌸', '🌹', '🌻', '🌽',
-    '🍀', '🍁', '🍄', '🍅', '🍆', '🍇', '🍈', '🍉', '🍊', '🍋', '🍌', '🍍', '🍎', '🍐', '🍑', '🍒', '🍓', '🍔', '🍕',
-    '🍗', '🍚', '🍞', '🍟', '🍠', '🍣', '🍦', '🍩', '🍪', '🍫', '🍬', '🍭', '🍯', '🍰', '🍳', '🍴', '🍵', '🍶', '🍷',
-    '🍸', '🍹', '🍺', '🍼', '🎀', '🎁', '🎂', '🎃', '🎄', '🎈', '🎉', '🎒', '🎓', '🎠', '🎡', '🎢', '🎣', '🎤', '🎥',
+    '🦋', '📟', '🌈', '🌊', '🎯', '🐋', '🌙', '🤔', '🌕', '⭐', '🎋', '🌰', '🌴', '🌵', '🌲', '🌸', '🌹', '🌻', '🌽',
```
Comment (hansieodendaal) [inline base_layer/common_types/src/emoji.rs:82]: **Nit:** Maybe add a reference to the YAT list?

**Review by hansieodendaal — DISMISSED** [review]
utACK

**Comment by AaronFeickert** [comment]
This will require corresponding updates to [RFC-0152](https://rfc.tari.com/RFC-0152_EmojiId) and the [Tari site](https://www.tari.com/lessons/05_emoji_id) to account for the new emoji alphabet (emojibet?).

**Review by AaronFeickert — APPROVED** [review]

**Comment by SWvheerden** [comment]
https://github.com/tari-project/rfcs/pull/145

**Pull request merged**